corehq/apps/export/tasks.py

```diff
@@ -96,6 +97,27 @@ def _cached_add_inferred_export_properties(sender, domain, case_type, properties
     inferred_schema.save()
 
 
-@task
+@task(soft_time_limit=SCHEMA_GENERATION_TIMEOUT)
```

Comment:
my assumption here is that `soft_time_limit` will start ticking from when we kick off the task vs. when the task actually starts. need to confirm